src/components/Tooltip/Tooltip.tsx

```diff
@@ -74,6 +81,18 @@ const Tooltip = ({
     }
   }, [])
 
+  useEffect(() => {
+    if (typeof window !== 'undefined' && tooltipRef.current) {
+      if (isOnShadowDOM && !root.getElementById('react-tooltip-styles')) {
+        const style = document.createElement('style')
+        style.id = 'react-tooltip-styles'
+        style.textContent = normalStyles
+
+        root.appendChild(style)
+      }
+    }
+  })
```

Comment:
I was not able to convert the CSS modules to text, so, I had to copy our current styles and export it as string to inject inside of shadow dom

Reply:
I'm able to do it on: https://github.com/ReactTooltip/react-tooltip/pull/1041